calcom/cal.com
# change datepicker styles to fix dark mode issue

## What does this PR do?

Fixes #8880

https://www.loom.com/share/13b0ddab647e4e9e8a1f5401127ce4de

**Environment**: Production

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

In Limit future bookings, use date-picker in both dark-mode and light-mode to test themes.

To see it in video :
https://www.loom.com/share/13b0ddab647e4e9e8a1f5401127ce4de
